src/constants/tokens.ts

```diff
@@ -32,11 +32,12 @@ export const OHM_TOKEN = new Token({
  * the token has been initialised to prevent
  * circular references during initialisation.
  */
-OHM_TOKEN.customPricingFunc = async () => {
-  const contract = OHM_DAI_LP_TOKEN.getEthersContract(NetworkId.MAINNET);
-  const [ohm, dai] = await contract.getReserves();
-  return new DecimalBigNumber(dai.div(ohm), 9);
-};
+// OHM_TOKEN.customPricingFunc = async () => {
+//   const contract = OHM_DAI_LP_TOKEN.getEthersContract(NetworkId.MAINNET);
+//   console.log("get Reservs xyz");
+//   const [ohm, dai] = await contract.getReserves();
+//   return new DecimalBigNumber(dai.div(ohm), 9);
+// };
```

Comment:
this is the code I want to remove. If you uncomment it you can see that it fires a bunch of times & I don't think it's used.

Reply:
if it meets your approval @sam-potter we can delete the commented code.

Reply:
It's being used to get the price of OHM in each bond. The app still works if you uncomment it, because it falls back to getting the price of the token from coingecko. However you'll notice that the discounts on each bond will differ slightly because coingecko's price is slightly different from the one calculated using reserves. 

So i guess whether we remove it depends on what price do you think is more accurate? Everywhere else in the UI we show the OHM price based on the reserves, so imo i think we should be consistent and keep that so that if a user does the maths themselves it'll add up

Reply:
Ah yes, you're right. Thank you